### PR TITLE
PIMOB-1518: Update the sample app as per new designs

### PR DESCRIPTION
**Pull request by chintan-soni-cko**
## Issue

[PIMOB-1518](https://checkout.atlassian.net/browse/PIMOB-1518)

## Proposed changes
1. Redesign the sample app to new UI
2. Added support for 3DS invocation from the headless SDK

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [X] Reviewers assigned
* [X] I have performed a self-review of my code and manual testing
* [X] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if applicable)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you choose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by maxim-nosov-cko — COMMENTED** [review]
There is no hint for the 3DS field, which makes it unclear what exactly should be used there.

File: example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/ThreedSecureActivity.kt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                componentActivity.finish()
+            }
+        }.show()
+    }
+
+    private fun invokeThreedSecureFlow(context: Context, url: String, viewGroup: ViewGroup) {
+        /**
+         * Creating instance of CheckoutApiClient
+         */
+        val checkoutApiClient = CheckoutApiServiceFactory.create(
```
Comment (maxim-nosov-cko) [inline example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/ThreedSecureActivity.kt:46]: Is it okay that we use headless SDK directly?
Why not to use frames instead?

**Review by maxim-nosov-cko — COMMENTED** [review]
The main screen should have a scrollable top-level container to cover small devices and avoid issues like those in the screenshot.
Screenshot_20221213_091448

File: example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/ThreedSecureActivity.kt (hunk 1 above)
Comment (chintan-soni-cko) [reply example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/ThreedSecureActivity.kt:46]: We can use but we need to add an unnecessary set to the mPaymentFormMediator and configuration for it. At the same time, we will provide merchant to set an example of how we can use headless SDK for 3DS support.

**Review by maxim-nosov-cko — APPROVED** [review]
Great job @chintan-soni-cko

**Pull request merged**